addons/point_of_sale/static/src/app/store/combo_configurator_popup/combo_configurator_popup.js

```diff
@@ -16,10 +16,41 @@ export class ComboConfiguratorPopup extends Component {
     setup() {
         this.pos = usePos();
         this.state = useState({
-            combo: Object.fromEntries(this.props.product.combo_ids.map((elem) => [elem.id, 0])),
+            combo: Object.fromEntries(this.props.product.combo_ids.map((combo) => [combo.id, 0])),
+            shouldShowCombo: Object.fromEntries(
```

Comment:
I'm not sure this should be in the state as this is never changed once initialized and we do not need this information to rerender the popup